spotinst_sdk2/clients/elastigroup/__init__.py

```diff
@@ -1494,13 +1501,13 @@ def update_elastigroup(self, group_update, group_id):
         """
         group = gcp_elastigroup.ElastigroupCreationRequest(group_update)
```

Comment:
Should have ElastigroupUpdateRequest class and the object of that class should be made.

Reply:
Done